prow/tide/tide.go

```diff
@@ -619,9 +619,21 @@ func (c *syncController) initSubpoolData(sp *subpool) error {
 	if err != nil {
 		return fmt.Errorf("error determining required presubmit prowjobs: %w", err)
 	}
+	// CloneURI is used for Gerrit to retrieve inrepoconfig, this is not used by
+	// GitHub at all. Blindly trust that it's identical among jobs.
```

Comment:
```suggestion
	// CloneURI is used by Gerrit to retrieve inrepoconfig; this is not used by
	// GitHub at all. Blindly trust that it's identical among jobs.
```

Nit: Explain what happens if it is not identical, or why blind trust here is OK. Also add a "because <reason X>" after "GitHub at all." part and fill in what <reason X> is in this case.